docs/percona.md

```diff
@@ -8,205 +8,363 @@ sidebar_label: Percona
+  
+- `openebs-device` - Using this option, it will create Kubernetes Local PVs using the block devices attached to the node. Select this option when you want to dedicate a complete block device on a node to a Cassandra node. You can customize which devices will be discovered and managed by OpenEBS using the instructions [here](https://docs.openebs.io/docs/next/ugndm.html). 
 
-5. **Launch and test Percona**:
+The Storage Class `openebs-device` has been chosen to deploy PXC in the Kubernetes cluster.
 
-   Create a file called `percona-openebs-deployment.yaml` and add content from `percona-openebs-deployment.yaml` given in the configuration details section. Run `kubectl apply -f percona-openebs-deployment.yaml` to deploy Percona application. For more information, see Percona documentation. In other way,  you can use stable Percona image with helm to deploy Percona in your cluster using the following command.
+### Install the Percona XtraDB Cluster operator
+```
+$ git clone -b v1.6.0 https://github.com/percona/percona-xtradb-cluster-operator
+$ cd percona-xtradb-cluster-operator
+$ kubectl apply -f deploy/bundle.yaml
+```
+Verify if the operator is running correctly
+```
+$ kubectl get pod
+NAME                                               READY   STATUS    RESTARTS   AGE
+percona-xtradb-cluster-operator-749b86b678-8f4q5   1/1     Running   0          23s
+```
 
-   ```
-   helm install --name my-release --set persistence.enabled=true,persistence.storageClass=openebs-cstor-disk stable/percona
-   ```
+Update Storage and Monitoring specification
 
-<br>
+In this document, we have made changes in the storage section for PXC and the monitoring section PMM.
 
-<hr>
+Changes done in the Storage section for PXC: 
 
-<br>
+Update Storage Class name and required storage parameters in deploy/cr.yaml. In this example, we have updated 
+```
+spec.pxc.volumeSpec.persistentVolumeClaim.storageClassName as “openebs-device” and 
+spec.pxc.volumeSpec.persistentVolumeClaim.resources.requests.storage as “100Gi”
+```
+Sample snippet:
+```
+    volumeSpec:
+      persistentVolumeClaim:
+        storageClassName: openebs-device
+        accessModes: [ "ReadWriteOnce" ]
+        resources:
+          requests:
+            storage: 100Gi
+```
+**Note:** Ensure you have 100Gi is attached with each Node. Else, provide the storage capacity as per the capacity of the available disk. 
 
-## Reference at openebs.ci
+Changes done in the Monitoring section for PMM:
 
-A [sample Percona server](https://www.openebs.ci/percona-cstor) at [https://openebs.ci](https://openebs.ci/)
+Enable monitoring service and server user name. In this example, we have updated 
+```
+spec.pmm.enabled as “true”
+spec.pmm.serverUser as “admin”  
+```
+The following is the sample snippet of PVC spec of Percona XtraDB where the Storage Class name and storage capacity has been changed.
```

Comment:
This statement does not match the content provided below

Reply:
PMM is the monitoring section in cr.yaml. So I hope it will get the context.

Reply:
Here it mentions  storage class and storage capacity, but the snippet is about pmm. 
Either we need to change the statement or provide the corresponding snippet 
```
The following is the sample snippet of PVC spec of Percona XtraDB where the Storage Class name and storage capacity has been changed.
```

Reply:
Done.